sunpy/visualization/animator/tests/test_basefuncanimator.py

```diff
@@ -29,15 +34,46 @@ def button_func1(*args, **kwargs):
                                                     (plt.figure(), True, [[button_func1], ["hi"]])))
 def test_base_func_init(fig, colorbar, buttons):
     data = np.random.random((3, 10, 10))
-    func = partial(update_plotval, data=data)
-    funcs = [func]
-    ranges = [(0, 3)]
+    func0 = partial(update_plotval, data=data)
+    func1 = partial(update_plotval, data=data*10)
+    funcs = [func0, func1]
+    ranges = [(0, 3), (4, 7)]
 
     tfa = FuncAnimatorTest(data, funcs, ranges, fig=fig, colorbar=colorbar,
                            button_func=buttons[0],
                            button_labels=buttons[1])
 
     tfa.label_slider(0, "hello")
+    assert tfa.sliders[0]._slider.label.get_text() == "hello"
+
+    tfa._set_active_slider(1)
+    assert tfa.active_slider == 1
+
+    fig = plt.figure()
+    event = mback.KeyEvent(name='key_press_event', canvas=fig.canvas, key='down')
+    tfa._key_press(event)
+    assert tfa.active_slider == 0
+
+    slide = tfa.active_slider
+    slider = widgets.Slider(fig.gca(), str(slide), ranges[slide][0], ranges[slide][1], valinit=0)
```

Comment:
I am not sure of the valinit thing. That way I was able to pass the tests anyway. Apparently, initial value of slider is default (0.5), and therefore gives assertion error in line 73.

Reply:
I commented `valint` out and it didn't seem to fail the test.

Reply:
Right, it doesn't. I was doing something wrong earlier perhaps?

Reply:
I was messing up with the sliders.